app/openshift/nodejs-mongodb-persistent.json

```diff
@@ -460,6 +460,13 @@
       "required": true
     },
     {
+      "name": "REGISTRY_PROJECT",
+      "displayName": "Registry project",
```

Comment:
In line of documenting this var a bit better, I'd suggest a more explicit display name... like "namespace in the registry where images are shared across environments" or something like that.

It might also be worth adding a comment in `host_vars/registry.example` around `project_name`

Reply:
@codificat will do.